class4.html

```diff
@@ -478,6 +482,34 @@
           </section>
  End Bootstrap -->
         
+
+        <!-- Google Fonts -->
+        <section>
+          <h2> Custom Fonts </h2>
+
+          <p> Head to this page: </p>
+          <p> <a href="https://www.google.com/fonts" target="_blank">https://www.google.com/fonts</a>
+          <ol>
+          <li>Find a font and select "add to collection."</li>
+          <li>Click "use", then select the font weights you'd like on your site. </li>
+          <li>Copy the &lt;link> snipet into your HTML &lt;head>. </li>
```

Comment:
If you use `&lt;` then use `&gt;` instead of  `>`  (keep html formatting consistent)